testConnectivity.py

```diff
@@ -275,25 +288,35 @@ def testConnectivity(args):
         printYellow("="*20)
 
         print("Checking GBT Communication (Before Programming GBTs)")
-        if not gbtCommIsGood(vfatBoard.parentOH.parentAMC, doReset=True, printSummary=args.debug, ohMask=args.ohMask):
+        if not gbtCommIsGood(vfatBoard.parentOH.parentAMC, doReset=True, printSummary=args.debug, ohMask=args.ohMask, gemType=args.gemType):
             printRed("Connectivity Testing Failed")
             printYellow("If Vmon = 8.0V then Imon must be 1.71 +/- 0.01A; if not the GBT's are not locking to the fiber link")
             return
 
         # Program GBTs
-        gbtConfigs = [
+        if args.gemType == "ge11":
+            gbtConfigs = [
                 "{0}/GBTX_OHv3c_GBT_0.txt".format(gbtConfigPath),
                 "{0}/GBTX_OHv3c_GBT_1.txt".format(gbtConfigPath),
                 "{0}/GBTX_OHv3c_GBT_2.txt".format(gbtConfigPath),
                 ]
+        elif args.gemType == "ge21":
+            gbtConfigs = [
+                "{0}/GBTX_GE21_OHv1_GBT_0_minimal_2019-01-24.txt".format(gbtConfigPath),
+                "{0}/GBTX_GE21_OHv1_GBT_1_minimal_2019-01-24.txt".format(gbtConfigPath),
```

Comment:
I would suggest to have these be generic symlinks rather then specific files as you will need to update this overtime (based on GE1/1 experience)